internal/commands/image/inspect.go

```diff
@@ -30,16 +30,17 @@ func muteDockerCli(dockerCli command.Cli) func() {
 func inspectCmd(dockerCli command.Cli) *cobra.Command {
 	var opts inspectOptions
 	cmd := &cobra.Command{
-		Use:   "inspect [APP_NAME] [OPTIONS]",
-		Short: "Shows metadata, parameters and a summary of the Compose file for a given application",
-		Example: `$ docker app inspect my-installed-app
-$docker app inspect my-app:1.0.0`,
+		Use:   "inspect [APP_IMAGE] [OPTIONS]",
```

Comment:
I think `APP_IMAGE` is mandatory so it should be: `inspect [OPTIONS] APP_IMAGE`

Reply:
Right. Corrected.

Reply:
I can't see the modification 🤔

Reply:
But I did the modification...